topics/setup.md

```diff
@@ -7,12 +7,12 @@ Before you begin [creating your first application](create-first-app.md) to work
    system supported by [Android Studio](https://developer.android.com/studio).  
    If you also want to write iOS-specific code and run an iOS application on a simulated or real device, use a Mac with a 
    macOS.  
-2. Install [Android Studio](https://developer.android.com/studio) – version 4.1 or higher.   
+2. Install [Android Studio](https://developer.android.com/studio)  – version 4.1 or higher.   
     You will use Android Studio for creating your multiplatform applications and running them on simulated or hardware devices.
 3. If you need to write iOS-specific code and run an iOS application, install [Xcode](https://apps.apple.com/us/app/xcode/id497799835) 
-    –  version 11.3 or higher.                                                                                                                                                                                                                                                                                                                          
+     –  version 11.3 or higher.                                                                                                                                                                                                                                                                                                                          
     Most of the time, Xcode will work in the background. You will use it to add Swift or Objective-C code to your iOS application.
-4. Update the Kotlin plugin to version 1.4.20 or higher.  
+4. If you have used your current installation of Android Studio before, update the Kotlin plugin to version 1.4.20 or higher.  
```

Comment:
Because if not, then you don't need to do anything. If you have just installed AS – everything is already up to date.